src/config/helmet.ts

```diff
@@ -95,7 +99,12 @@ export const webchatHelmetConfiguration: HelmetOptions = {
         "https://*.smartagent.app",
         "https://*.ruxit.com",
         "https://*.dynatrace.com",
+        "https://participant.connect.eu-west-2.amazonaws.com",
+        (req: Request, res: Response): string =>
+          `'${res.locals.missionLabWebSocketAddress}'`,
```

Comment:
There has been an update to the recommendations, to add https://api.rollbar.com to the allowlist

Reply:
Ah okay, yes happy to add.